stripe/api_requestor.py

```diff
@@ -286,16 +311,20 @@ def request_headers(self, api_key, method):
 
     def request_raw(
         self,
-        method,
-        url,
-        params=None,
-        supplied_headers=None,
-        is_streaming=False,
-    ):
+        method: str,
+        url: str,
+        params: Optional[Mapping[str, Any]] = None,
+        supplied_headers: Optional[Mapping[str, str]] = None,
+        is_streaming: bool = False,
+    ) -> Tuple[object, int, Mapping[str, str], str]:
         """
         Mechanism for issuing an API call
         """
 
+        supplied_headers_: Optional[Dict[str, str]] = (
```

Comment:
```suggestion
        supplied_header_dict: Optional[Dict[str, str]] = (
```

Reply:
Great idea, done in [Address code review](https://github.com/stripe/stripe-python/pull/1088/commits/067ab1bfd6a74acd7cb4bb8c4f7e878a2d0e4661)